### ISSUE TITLE
[Question] How do I see all types available under a namespace?

### ISSUE THREAD

**Issue by Kharms** [mvantellingen/python-zeep#481]
How do I see all types available under a namespace?

Based on example in documentation http://docs.python-zeep.org/en/master/datastructures.html

`from zeep import Client

client = Client('http://my-enterprise-endpoint.com')
order_type = client.get_type('ns0:Order')
order = order_type(number='1234', price=99)
client.service.submit_order(user_id=1, order=order)`

How do see all available types under 'ns0:' using python?

**Comment by mvantellingen** [comment]
python -mzeep <wsdl>

**Comment by Kharms** [comment]
I don't think I communicated my question well, let me rephrase:

How would I navigate `client.wsdl` to accomplish that? Is there functionality similar to `soup.findAll('tag')` from [beautifulsoup](https://www.crummy.com/software/BeautifulSoup/bs3/documentation.html#The%20basic%20find%20method:%20findAll(name,%20attrs,%20recursive,%20text,%20limit,%20**kwargs))? 

BeautifulSoup example:
```
soup.findAll(['title', 'p'])
# [<title>Page title</title>, 
#  <p id="firstpara" align="center">This is paragraph <b>one</b>.</p>, 
#  <p id="secondpara" align="blah">This is paragraph <b>two</b>.</p>]
```

What I'd like to accomplish:
```
client.wsdl.types.find_all(namespace='ns0')
# <ns0:type1>
#  <ns0:type2>
#  <ns0:type3>
...
```

**Comment by zinob** [comment]
I know this is an ancient post  (hi [DenverCoder9](https://xkcd.com/979/)), but I just ran into this problem the second time and actually felt like i needed to find a solution this time, so i checked the [`__main__` code](https://github.com/mvantellingen/python-zeep/blob/master/src/zeep/wsdl/wsdl.py#L123) for what is run when you call zeep on a wsdl. Apparently you can find all the type-objects in `client.wsdl.types.types`
What the WSDL-formatting tool does is roughly equivalent to `[i.signature(schema=client.wsdl.types) for i in client.wsdl.types.types]`
